tests/karma.js

```diff
@@ -1064,6 +1064,204 @@ beforeAll(function (done) {
           body: unauthorizedXmlResponseBody
         }
       }))
+    .then(() =>
+      provider.addInteraction({
+        uponReceiving: 'Share a file with a link', // Try to share with wrong password
```

Comment:
I think we need to specify in the request name rather than in the comment that this is an unauthorized request. there are/may be similar authorized requests as well and it might create confusion or duplications in the request name